desktop_version/src/Input.cpp

```diff
@@ -1332,6 +1331,7 @@ SDL_assert(0 && "Remove open level dir");
 
 					if (game.currentmenuoption == 4)
 					{
+                        music.playef(11, 10);
```

Comment:
Weird tab here, other than that LGTM!

Reply:
Most of this file is spaces, but a few places are tabs: this is an area that uses tabs (which I didn't notice). Fixed now.